src/XeroPHP/Application/PartnerApplication.php

```diff
@@ -8,8 +8,8 @@ class PartnerApplication extends Application
 {
     protected static $_type_config_defaults = [
         'xero' => [
-            'site'     => 'https://api-partner.network.xero.com',
-            'base_url' => 'https://api-partner.network.xero.com',
+            'site'     => 'https://api.xero.com',
+            'base_url' => 'https://api.xero.com',
```

Comment:
Thanks for update @SidneyAllen - I was aware of it but have just implemented it int he `2.0` branch as it's nearing being ready.  These settings actually only override the defaults, so if they're no different to the other APIs, the keys can be removed entirely.